### DISCUSSION TITLE
Process Hacker Kernel Mode driver is now revoked on Windows 11 Build 22621.521

### DISCUSSION THREAD

**Discussion by Aholicknight** [winsiderss/systeminformer#1384] [Questions / Answers] [answered]
![image](https://user-images.githubusercontent.com/7843719/191827853-50dc0e59-eb4b-4894-abbc-bc9de5fb89af.png)
I recently updated to the latest version of Windows 11 version 22621.521 and when I opened Process Hacker I get the following message:

![image](https://user-images.githubusercontent.com/7843719/191827953-df03df26-4214-4577-b580-e28fc7c7ce68.png)

This message never appeared before on the earlier version of Windows 11 I had and only shows now. Is there anyway to fix this? Would I have to compromise my computers security and disable driver enforcement just so I can get kernel mode access for process hacker or will a update come out soon? Thanks.

https://user-images.githubusercontent.com/7843719/191828659-adc30e11-706c-4dfd-b56e-e9134eb7b598.mp4

And for some reason the `kprocesshacker.sys` driver is still signed so I do not know why Process Hacker says it's revoked.

**Comment by dmex** [comment, marked as answer]
> And for some reason the kprocesshacker.sys driver is still signed so I do not know why Process Hacker says it's revoked.

The certificate hasn't been revoked. That's the fake error code Microsoft uses when they block drivers via HVCI.

> s there anyway to fix this?

https://github.com/winsiderss/systeminformer/issues/1142#issuecomment-1148635917

**Reply by Aholicknight** [reply]
Thank you!
